inc/db.function.php

```diff
@@ -1736,6 +1736,10 @@ function getEntitiesRestrictRequest($separator = "AND", $table = "", $field = ""
    if (is_array($value)) {
       $query .= " IN ('" . implode("','", $value) . "') ";
    } else {
+      if (strlen($value) == 0 && !isset($_SESSION['glpiactiveentities_string'])) {
+         //set root entity if not set
+         $value = 0;
+      }
       if (strlen($value) == 0) {
```

Comment:
When `glpiactiveentities_string` does not exists; `$value` ist set to `0`; we therefore pass in the else part and work with root entity only.